### PR TITLE
Resolve issue #229

### PR DESCRIPTION
**Pull request by vdakalov**

### CODE REVIEW

**Comment by vdakalov** [comment]
Waiting for accept https://github.com/mcollina/bloomrun/pull/67

File: test/hemera/unsubscribing.spec.js [hunk 1]
```diff
@@ -212,6 +212,15 @@ describe('Unsubscribe NATS topic', function() {
         }
       )
 
+      hemera.add(
```
Comment (StarpTech) [inline test/hemera/unsubscribing.spec.js]: What are you trying to test?

Comment (vdakalov) [reply test/hemera/unsubscribing.spec.js]: how `removeAll` function works with wildcard topic

Comment (vdakalov) [reply test/hemera/unsubscribing.spec.js]: ...it should remove all patterns event if they is wildcard topic

**Comment by StarpTech** [comment]
Closed due to inactivity.

**Pull request closed without merging**